web/profiles/joinup/config/install/filter.format.full_html.yml

```diff
@@ -6,8 +6,6 @@ dependencies:
 name: 'Full HTML'
 format: full_html
 weight: 1
-roles:
-  - administrator
```

Comment:
Same here, we shouldn't be touching the filter formats in scope of this ticket. Let's revert this.